docs/topics/Gateway.md

```diff
@@ -177,6 +177,115 @@ If successful, the gateway will respond by replaying all missed events in order,
+  - GUILD_BAN_ADD
+  - GUILD_BAN_REMOVE
+
+GUILD_EMOJIS (1 << 3)
+  - GUILD_EMOJIS_UPDATE
+
+GUILD_INTEGRATIONS (1 << 4)
+  - GUILD_INTEGRATIONS_UPDATE
+
+GUILD_WEBHOOKS (1 << 5)
+  - WEBHOOKS_UPDATE
+
+GUILD_INVITES (1 << 6)
+  - INVITE_CREATE
+  - INVITE_DELETE
+
+GUILD_VOICE_STATES (1 << 7)
+  - VOICE_STATE_UPDATE
+
+GUILD_PRESENCES (1 << 8)
+  - PRESENCE_UPDATE
+
+GUILD_MESSAGES (1 << 9)
+  - MESSAGE_CREATE
+  - MESSAGE_UPDATE
+  - MESSAGE_DELETE
+
+GUILD_MESSAGE_REACTIONS (1 << 10)
+  - MESSAGE_REACTION_ADD
+  - MESSAGE_REACTION_REMOVE
+  - MESSAGE_REACTION_REMOVE_ALL
+  - MESSAGE_REACTION_REMOVE_EMOJI
+
+GUILD_MESSAGE_TYPING (1 << 11)
+  - TYPING_START
+
+DIRECT_MESSAGES (1 << 12)
+  - CHANNEL_CREATE
+  - MESSAGE_CREATE
+  - MESSAGE_UPDATE
+  - MESSAGE_DELETE
+  - CHANNEL_PINS_UPDATE
+
+DIRECT_MESSAGE_REACTIONS (1 << 13)
+  - MESSAGE_REACTION_ADD
+  - MESSAGE_REACTION_REMOVE
+  - MESSAGE_REACTION_REMOVE_ALL
+  - MESSAGE_REACTION_REMOVE_EMOJI
+
+DIRECT_MESSAGE_TYPING (1 << 14)
+  - TYPING_START
+```
+
+Any [events not defined in an intent](#DOCS_TOPICS_GATEWAY/commands-and-events-gateway-events) are considered "passthrough" and will always be sent to you. If you specify an `intent` value in your `IDENTIFY` payload that is invalid, the socket will close with a [`4012` close code](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/gateway-gateway-close-event-codes). An invalid intent value can be one of:
+
+- A miscalculated bit flag
+- Specifying a privileged intent for which you have not opted in for your bot
+- Specifying a privileged intent for which you have not been whitelisted, and your bot is in more than the current limit for unwhitelisted access
+
+### Privileged Intents
+
+> danger
+> As of <date>, Gateway Intents are optional in v6 **except for `GUILD_PRESENCES`**. If you wish to get `PRESENCE_UPDATE` events, you **must** declare the intent. Otherwise, you will no longer receive `PRESENCE_UPDATE` events on the v6 gateway after <date>
+
+> warns
+> The current limit for specifying privileged intents without whitelisting is 100 guilds.
+
+Some intents require whitelisting once your bot scales beyond a certain size. Those intents are:
+
+- `GUILD_PRESENCES`
+
+The current limit for specifying these privileged intents without being whitelisted is 100 guilds. That means you may specify the above intents without restriction until you hit the current limit, at which point your bot will not be able to join any more guilds without either being whitelisted or dropping the privileged intents for which you have not been whitelisted.
+
+In order to specify these intents in your `IDENTIFY` payload, you must first go to your application in the Developer Portal and enable the toggle for the whitelisted intent you wish to use.
+
+> danger
+> **Enabling a privileged intent without being whitelisted will limit your bot to joining 100 total guilds until you are whitelisted.** To get whitelisted, please fill out this form: <form_link_goes_here>
```

Comment:
There needs to be clarity on the process and criteria of being whitelisted, if the aim is to stop people making massive bots that scrape data to sell then there will obviously be some requirements to be whitelisted because if you just whitelisted everyone who submitted the form then the problem this aims to solve wouldn't be fixed?

Perhaps an extra section within the documentation to cover how to get whitelisted and what requirements need to be fulfilled.